client/flutter/lib/l10n/intl_pt.arb

```diff
@@ -0,0 +1,26 @@
+{
+  "worldHealthOrganization": "Organização Mundial da Saúde",
+  "learnMore": "Saiba mais",
+  "protectYourself": "Proteja-se",
+  "checkYourHealth": "Cheque sua saúde",
+  "localMaps": "Mapas Locais",
+  "shareTheApp": "Compartilhe o App",
+  "healthCheck": "Exame de Saúde",
+  "feelingDistressed": "Sentindo-se angustiado?",
+  "washHands": "Lave suas mãos frequentemente com sabão e água corrente",
+  "cougningAndSneezing": "Ao tossir ou espirar cubra sua boca e nariz com o cotovelo flexionado ou um lenço de papel",
+  "throwAwayTissue": "Jogue o lenço de papel em uma lixeira fechada imediatamente após o uso",
+  "washHandsFrequently": "Lave as mãos frequentemente",
+  "socialDistancing": "Evite o contato próximo e mantenha distância fisíca",
+  "seekMedicalCare": "Procure por atendimento médico logo se tiver febre, tosse e dificuldade para respirar",
+  "infoForEveryone": "Informação para Todos",
+  "infoForParents": "Informação para os Pais",
+  "childStress": "As crianças podem responder ao estresse de diferentes maneiras como ser mais apegado, ansioso, distante, irritado ou agitado, urinar na cama etc.\nResponda às reações do seu filho de forma solidária, ouça as suas preocupações e dê a eles mais amor e atenção.",
+  "childLove": "As crianças precisam de amor e atenção dos adultos em tempos difíceis. Dedique a eles mais tempo e atenção.\nLembre-se de ouvir seus filhos, falar gentilmente e os tranquilize.\nSe possível, crie oportunidades para a criança brincarer e relaxar.",
+  "childDistance": "Tente manter as crianças próximas dos seus pais e família e evite as separar dos seus responsáveis na medida do possível. Se ocorrer separação (por exemplo, hospitalização), assegure contato regular (por exemplo, por telefone) e tranquilização.",
+  "childRoutines": "Mantenha rotinas regulares e horários o máximo possível, ou ajude a criar novos em um novos ambientes, incluindo escola/aprendizagem, bem como tempo para brincar com segurança e relaxar.",
+  "childFacts": "Forneça fatos sobre o que aconteceu, explique o que está acontecendo e forneça informações claras sobre como reduzir os riscos de serem infectados pela doença em palavras que possam entender dependendo da idade.\nIsso inclui também fornecer informações sobre o que poderia acontecer de uma maneira reconfortante (por exemplo, um membro da família e/ou a criança pode começar a não se sentir bem e precisar ter que ir ao hospital por algum tempo para que os médicos possam ajudá-los a melhorar).",
+  "whoMythBusters": "WHO Caçadores de mitos",
```

Comment:
Since "World Health Organization" in Portuguese translates to "Organização Mundial da Saúde", maybe we should use "OMS" instead of "WHO"?

```suggestion
  "whoMythBusters": "OMS Caçadores de mitos",
```